### PR TITLE
Disable minus button when page count is 0

### LINKED ISSUE #892: Disable the "-" button when you can't use it on pages
When users can't remove a page, we should hide the - button and show it when it applies again.

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30928

**Changes**
- Disable the `-` button when the number of selected pages for that particular page type is 0.

closes #892

### CODE REVIEW

File: src/client/src/components/SelectableCard/styles.module.css [hunk 1]
```diff
@@ -161,3 +161,8 @@
   display: flex;
   font-size: 14px;
 }
+
+.disabledOverlay {
```
Comment (jcoc611-microsoft) [inline src/client/src/components/SelectableCard/styles.module.css]: Shouldn't there just be a style for `button:disabled`

Comment (streetcoder123) [reply src/client/src/components/SelectableCard/styles.module.css]: Perfectly makes sense.

**Review by jcoc611-microsoft — DISMISSED** [review]

**Review by SahilTara — DISMISSED** [review]

**Comment by jasmingoh** [comment]
Just to make sure, is it intended that the cursor is a pointer for the disabled minus button?

**Comment by streetcoder123** [comment]
@jasmingoh That's a good observation. I think in an ideal scenario disabled buttons should not have cursor pointers. Is that right @jcoc611-microsoft

**Comment by streetcoder123** [comment]
@jasmingoh I have made the change to default the cursor for disabled buttons

**Review by jsondoo — APPROVED** [review]

**Review by jasmingoh — APPROVED** [review]

**Review by piotrmark — APPROVED** [review]

**Pull request merged**